R/html-tweak.R

```diff
@@ -23,7 +23,7 @@ tweak_anchors <- function(html, only_contents = TRUE) {
     gsub(".", "-", ., fixed = TRUE)
   purrr::walk2(toc_nav, hrefs, ~ (xml2::xml_attr(.x, "href") <- .y))
 
-  headings <- xml2::xml_find_first(sections, ".//h1|h2|h3|h4|h5")
+  headings <- xml2::xml_find_first(sections, ".//h2|h3|h4|h5")
```

Comment:
It was a bit tricky to get the bottom border with the new heading/anchor structure, and I was thinking a direct link to h1 is maybe not that useful? Now, one might find the interface less predictable. :thinking: